settings.lua

```diff
@@ -27,8 +27,22 @@ data:extend({
     default_value = true,
     order = "2",
   },
+  {
+    type = "int-setting",
+    name = "cube-cubecam-scale",
```

Comment:
did you forget to add a change to the locale file? (I did see it looked right in the video you posted.)

Reply:
yep forgot to copy that over